src/Processors/Formats/Impl/OneFormat.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <Processors/Formats/IInputFormat.h>
+#include <Processors/Formats/ISchemaReader.h>
+#include <DataTypes/DataTypesNumber.h>
+
+namespace DB
+{
+
+class OneInputFormat final : public IInputFormat
+{
+public:
+    OneInputFormat(const Block & header, ReadBuffer & in_);
```

Comment:
Two questions:

1. Shouldn't we also have OneOutputFormat for consistency?
2. Is it possible to avoid even opening the file?